statsmodels/multivariate/factor.py

```diff
@@ -346,7 +348,8 @@ def score(self, par):
 
         Returns
         -------
-        score : ndarray
+        ndarray
```

Comment:
I don't like these kind of changes.
In the old version, I can see with one look what it is, score or loglike,
In the new docstring, you have to read a sentence.

In short docstrings like this it doesn't make much difference because the function name is not far away.
I long docstring, there can be a lot of lines between function signature and Return docstring

Reply:
The other advantage is that it provides a hint about a name to use for the return